docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# How To Contribute
+
+## Some Ways to Contribute
+- Getting started with [L3AF](https://wiki.lfnetworking.org/display/L3AF/Getting+Started+with+L3AF)
+- Report potential bugs
```

Comment:
Some of these have sections below and some don't.  Is this supposed to be a table of contents that links to sections below?